flatgfa-py/src/lib.rs

```diff
@@ -44,6 +51,10 @@ impl Store {
             Store::File(ref mmap) => file::view(mmap),
         }
     }
+    fn py_depth(&self) {
+        let gfa = self.view();
+        depth(&gfa)
+    }
```

Comment:
This looks like an unrelated change. Please remove it from this PR so we can focus on the matter at hand: the GAF parser workflow.